packages/schemas/src/seeds/sign-in-experience.ts

```diff
@@ -1,14 +1,17 @@
 import { Language } from '@logto/phrases-ui';
+import { generateDarkColor } from '@logto/shared';
 
 import { CreateSignInExperience, SignInMode } from '../db-entries';
 import { BrandingStyle, SignInMethodState } from '../foundations';
 
+const defaultPrimaryColor = '#6139F6';
+
 export const defaultSignInExperience: Readonly<CreateSignInExperience> = {
   id: 'default',
   color: {
-    primaryColor: '#6139F6',
-    isDarkModeEnabled: false,
-    darkPrimaryColor: '#8768F8',
+    primaryColor: defaultPrimaryColor,
+    isDarkModeEnabled: true,
+    darkPrimaryColor: generateDarkColor(defaultPrimaryColor),
```

Comment:
this should be empty if the above is false

Reply:
`darkPrimaryColor` is non-null, so need to generate a dark color even is disabled.